pkgs/applications/editors/neovim/utils.nix

```diff
@@ -184,4 +186,9 @@ in
 {
   inherit makeNeovimConfig;
   inherit legacyWrapper;
+
+  inherit (callPackage ./build-neovim-plugin.nix {
+    inherit (vimUtils) buildVimPluginFrom2Nix toVimPlugin;
+    inherit buildLuarocksPackage;
+  }) buildNeovimPluginFrom2Nix;
```

Comment:
while it's not a problem here this is generally not a great pattern. the `x` in `inherit (x) ys ...` is evaluated indepdently for every `y`, so adding something here would evaluate the whole thing many times

Reply:
ha I had no idea thanks, I've removed the outer "inherit".